src/commandHandlers/bio.ts

```diff
@@ -13,7 +13,7 @@ import { log } from '../logger';
 export const command = async (arg: string, embed: MessageEmbed, message: Message) => {
     const args = arg.split('||');
     const mention = message.mentions.users.first();
-
+    const args[0] = args[0].toLowerCase();
```

Comment:
Sorry I probably didn’t explain myself well. You will need to create a new const, something like...

`const field = args[0].toLowerCase();`

Then use `field` throughout the code instead of `args[0]`

Reply:
Okay noted!